native2ika/processor_test.go

```diff
@@ -41,3 +42,60 @@ func TestRun(t *testing.T) {
 	assert.Equal(t, 3, len(retrievedSignRequests))
 	assert.NotNil(t, retrievedSignRequests[0].FinalSig)
 }
+
+func TestRun_NoPendingRequests(t *testing.T) {
```

Comment:
Let's convert these functions to table tests.

Reply:
Converted it into table tests, however i am not sure it fits here, let me know what you think